configure.ml

```diff
@@ -80,16 +80,46 @@ let expand start finish input output =
     exit 1;
   end
 
+let output_file filename lines =
+  let oc = open_out filename in
+  let lines = List.map (fun line -> line ^ "\n") lines in
+  List.iter (output_string oc) lines;
+  close_out oc
+
+let cc c_program =
+  let c_file = Filename.temp_file "configure" ".c" in
+  let o_file = c_file ^ ".o" in
+  output_file c_file c_program;
+  let found = Sys.command (Printf.sprintf "cc -Werror -c %s -o %s %s" c_file o_file "2>/dev/null") = 0 in
```

Comment:
`Printf.sprintf "cc -Werror -c %s -o %s 2>/dev/null" c_file o_file` - not sure it is better but more standard.